src/components/Helper.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div>
+    <sui-modal :open="open" v-on:changed="close">
+      <sui-modal-header>Help page</sui-modal-header>
+      <sui-modal-content image>
+        <sui-image
+          wrapped
+          :size="helpInformations[pageIndex].imageSize || 'medium'"
+          :src="helpInformations[pageIndex].imgUrl"
+        />
+        <!-- <img wrapped size="large" :src="imgUrl" /> -->
+        <sui-modal-description>
+          <sui-header>{{ helpInformations[pageIndex].title }}</sui-header>
+          <p
+            v-for="(text, index) in helpInformations[pageIndex].texts"
+            :key="index"
+            class="explanationText"
+          >
+            {{ text }}
+          </p>
+        </sui-modal-description>
+      </sui-modal-content>
+      <sui-segment aligned="right">
+        <sui-progress attached top :percent="pageRatio" color="green" />
+        <sui-button @click.native="prev">prev</sui-button>
+        <sui-button
```

Comment:
Would it make sense for this button to be blue (class `primary` on other views)? I think the 'next'  button does not have a strong call to action as-is:
![image](https://user-images.githubusercontent.com/87124/109417507-3d10e380-79c4-11eb-836b-2590bb6e8afc.png)

Reply:
Good idea